docs/references/backend/verify-token.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: verifyToken()
+description: Use Clerk's Backend SDK to verify a token signature.
+---
+
+{/* JS page: https://github.com/clerk/javascript/blob/843e96e505cd91d676f0736e24465726b55014a7/packages/backend/src/tokens/verify.ts#L13 */}
+
+# `verifyToken()`
```

Comment:
I feel like we might want to call out that this is a lower-level method and intended for more advanced use-cases, in most cases we want to recommend that people use `authenticateRequest()` instead if they are securing a more traditional API.